test_runner/regress/test_local_file_cache.py

```diff
@@ -0,0 +1,74 @@
+import os
+import random
+import threading
+import time
+from typing import List
+
+from fixtures.neon_fixtures import NeonEnv
+from fixtures.utils import query_scalar
+
+
+def test_local_file_cache_unlink(neon_simple_env: NeonEnv):
```

Comment:
From the test name I would expect that it performs LFC disable, or something like that, but I don't see it in the test below.
Is this a legacy name or am I missing something?

Reply:
This test checks corruption of LFC file (when for some reasons it disappears). It is simulated by renaming LFC file.